### PR TITLE
feat: dev/inheritance

### PR DESCRIPTION
**Pull request by baylagas**
added files and folder for inheritance

### CODE REVIEW

**Comment by baylagas** [comment]
this is still not finished, i left it in minute: 2:50:17

**Comment by baylagas** [comment]
Hi @jreyben  i have added the Ortus-Folder to the project

**Comment by lmajano** [comment]
@baylagas why are you building the components in tags instead of scripts?

File: Ortus-Folder/Inheritance/Musician.cfc [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+        <cfif arguments.myAge lt 1 or arguments.myAge gt 100>
+            <cfthrow message="age is invalid!">
+            <cfabort>
+        <cfelse>
+            <cfset variables.age = arguments.myAge>
+        </cfif>
+    </cffunction>
+
+    <cffunction  name="getPaid" access="public">
```
Comment (lmajano) [inline Ortus-Folder/Inheritance/Musician.cfc:37]: Components should not produce output. It breaks encapsulation.  It just return the data and then the caller should produce the output?

File: Ortus-Folder/Inheritance/Musician.cfc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <p>his age is #variables.age#</p>
+        </cfoutput>
+    </cffunction>
+
+    <cffunction  name="setAge">
+        <cfargument  name="myAge">
+
+        <cfif arguments.myAge lt 1 or arguments.myAge gt 100>
+            <cfthrow message="age is invalid!">
+            <cfabort>
```
Comment (lmajano) [inline Ortus-Folder/Inheritance/Musician.cfc:31]: I fyou do a throw, no abort is necessary

**Comment by lmajano** [comment]
Is this coming from a training?

**Comment by baylagas** [comment]
@lmajano Hello yes, its part of the way nolan is taking us

File: Ortus-Folder/Inheritance/Musician.cfc (hunk 2 above)
Comment (baylagas) [reply Ortus-Folder/Inheritance/Musician.cfc:31]: awesome, note taken

File: Ortus-Folder/Inheritance/Musician.cfc (hunk 1 above)
Comment (baylagas) [reply Ortus-Folder/Inheritance/Musician.cfc:37]: awesome note taken too

**Comment by lmajano** [comment]
Is this a video training?

**Comment by baylagas** [comment]
@lmajano that is correct

**Pull request merged**